Utility/ResetToolbars.FCMacro

```diff
@@ -0,0 +1,45 @@
+# Reset Toolbars position
```

Comment:
Please move the macro to `PureGui/GuiResetToolbars.FCMacro`.

Reply:
Based on this, I have changed the name to GuiResetToolbars.